module/Decision/src/Mapper/Member.php

```diff
@@ -50,10 +50,12 @@ public function searchByName(
             ->addScalarResult('generation', 'generation', 'integer');
 
         $sql = <<<QUERY
-            SELECT `lidnr`, CONCAT_WS(' ', `firstName`, IF(LENGTH(`middleName`), `middleName`, NULL), `lastName`) as `fullName`, `generation`
+            SELECT `lidnr`,
+                    CONCAT_WS(' ', `firstName`, IF(LENGTH(`middleName`), `middleName`, NULL), `lastName`) as `fullName`,
+                    `generation`
             FROM `Member`
-            WHERE CONCAT(LOWER(`firstName`), ' ', LOWER(`lastName`)) LIKE :name
-            OR CONCAT(LOWER(`firstName`), ' ', LOWER(`middleName`), ' ', LOWER(`lastName`)) LIKE :name
+            WHERE CONCAT_WS(' ', `firstName`, IF(LENGTH(`middleName`), `middleName`, NULL), `lastName`) LIKE :name
```

Comment:
This will prevent users from searching for people who have a middle name without also providing said middle name (partially or complete). Let's say we have a member `Jane van Doe`, and Alex is trying to find their profile yet they only now their first and last name, searching for `Jane D(oe)` will not yield any results (with the assumption that their are many `Jane`s).